src/Integration/Binding/RoslynConfigProjectBinder.cs

```diff
@@ -0,0 +1,26 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarLint.VisualStudio.Integration.Binding
+{
+    internal class RoslynConfigProjectBinder : IConfigProjectBinder
```

Comment:
I find the use of `Roslyn` in the name of this class and the associated binder class a bit confusing since they don't reference any of the `Roslyn` APIs.
Why `Roslyn`? Because the projects support Roslyn analyzers?

Reply:
Yep, to differentiate between roslyn / dotnet and non-roslyn projects. I'd expect roslyn ones to have a ruleset and parameters. How do you think it should be called?